### ISSUE TITLE
compiler stuck in Infinite loop.

### ISSUE THREAD

**Issue by MountainLabz** [GabeRundlett/gvox_engine#16]
When I try to create the cmake files, the compiler gets stuck in a loop installing the dependencies over and over again. something about finding an external library ninja 1.11 or something to that effect and then everything following it repeats over and over again.

**Comment by GabeRundlett** [comment]
there are many dependencies. Are you sure its not just taking a long time to install each dependency?

**Comment by MountainLabz** [comment]
No, I'm pretty sure its looping, or at least the display is looping, it says its doing x and then is followed by a bunch of text, then its says its doing x (same as before) followed by the same text as before. Its definitely not something wrong with your engine, I'm just curious what perhaps I need to do to get it working.

**Comment by GabeRundlett** [comment]
I'm sorry, I have never experienced this and can't reproduce it, so I'm not sure what the cause or solution are

**Comment by MountainLabz** [comment]
Thats alright.
